tutorialModels/ToyCarSpecular/glTF/ToyCarSpecular.gltf

```diff
@@ -112,7 +112,7 @@
                         "index": 8,
                         "texCoord": 0
                     },
-                    "specularColorTexture" : {
+                    "specularTexture" : {
```

Comment:
This looks weird. Why does this `specularTexture` block appear twice?

Reply:
Yes. I think your point is correct.
This model is effectively a copy from glTF-Sample-Models. You may need to have the original model modified.

https://github.com/KhronosGroup/glTF-Sample-Models/pull/268

Reply:
It was corrected in the following PR.
https://github.com/cx20/gltf-test/pull/390